enable-disk-spill-encrypt.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 为 TiDB 落盘文件开启加密
```

Comment:
@TomShawn Hi, can you help to check what's the alias of this file?

Reply:
This is a new document, so it does not need an alias.